altair/expr/__init__.py

```diff
@@ -117,6 +117,115 @@ class expr(_ExprRef, metaclass=_ExprMeta):
+    .. _Math.sqrt:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/sqrt
+    .. _Math.tan:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/tan
+    .. _normal (Gaussian) probability distribution:
+       https://en.wikipedia.org/wiki/Normal_distribution
+    .. _cumulative distribution function:
+       https://en.wikipedia.org/wiki/Cumulative_distribution_function
+    .. _probability density function:
+       https://en.wikipedia.org/wiki/Probability_density_function
+    .. _log-normal probability distribution:
+       https://en.wikipedia.org/wiki/Log-normal_distribution
+    .. _continuous uniform probability distribution:
+       https://en.wikipedia.org/wiki/Uniform_distribution_(continuous
+    .. _*unit*:
+       ../api/time/#time-units
+    .. _JavaScript's String.replace:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace
+    .. _d3-format specifier:
+       https://github.com/d3/d3-format/
+    .. _*units*:
+       ../api/time/#time-units
+    .. _timeUnitSpecifier API documentation:
+       ../api/time/#timeUnitSpecifier
+    .. _timeFormat:
+       #timeFormat
+    .. _utcFormat:
+       #utcFormat
+    .. _d3-time-format specifier:
+       https://github.com/d3/d3-time-format/
+    .. _TimeMultiFormat object:
+       ../types/#TimeMultiFormat
+    .. _UTC:
+       https://en.wikipedia.org/wiki/Coordinated_Universal_Time
+    .. _JavaScript's RegExp:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp
+    .. _RGB:
+       https://en.wikipedia.org/wiki/RGB_color_model
+    .. _d3-color's rgb function:
+       https://github.com/d3/d3-color#rgb
+    .. _HSL:
+       https://en.wikipedia.org/wiki/HSL_and_HSV
+    .. _d3-color's hsl function:
+       https://github.com/d3/d3-color#hsl
+    .. _CIE LAB:
+       https://en.wikipedia.org/wiki/Lab_color_space#CIELAB
+    .. _d3-color's lab function:
+       https://github.com/d3/d3-color#lab
+    .. _HCL:
+       https://en.wikipedia.org/wiki/Lab_color_space#CIELAB
+    .. _d3-color's hcl function:
+       https://github.com/d3/d3-color#hcl
+    .. _W3C Web Content Accessibility Guidelines:
+       https://www.w3.org/TR/2008/REC-WCAG20-20081211/#contrast-ratiodef
+    .. _continuous color scheme:
+       ../schemes
+    .. _geoArea:
+       https://github.com/d3/d3-geo#geoArea
+    .. _path.area:
+       https://github.com/d3/d3-geo#path_area
+    .. _geoBounds:
+       https://github.com/d3/d3-geo#geoBounds
+    .. _path.bounds:
+       https://github.com/d3/d3-geo#path_bounds
+    .. _geoCentroid:
+       https://github.com/d3/d3-geo#geoCentroid
+    .. _path.centroid:
+       https://github.com/d3/d3-geo#path_centroid
+    .. _window.screen:
+       https://developer.mozilla.org/en-US/docs/Web/API/Window/screen
```

Comment:
These are included so that the links resolve properly when they are used in the expression class methods below.